gpii/node_modules/flowManager/test/js/PrivacyFilterTests.js

```diff
@@ -0,0 +1,215 @@
+/**
+ * GPII Flow Manager Privacy Filter Tests
+ *
+ * Copyright 2015 OCAD University
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * The research leading to these results has received funding from the European Union's
+ * Seventh Framework Programme (FP7/2007-2013)
+ * under grant agreement no. 289016.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+/* global jqUnit, jQuery */
+
+"use strict";
+
+var fluid = fluid || require("infusion");
+
+(function ($) {
+
+    var gpii = fluid.registerNamespace("gpii");
+
+    fluid.registerNamespace("gpii.tests.flowManager.privacy.testdata");
+
+    gpii.tests.flowManager.privacy.testdata.payload = $.ajax({
+        type: "GET",
+        url: "../data/matchmaker.output.payload.json",
+        async: false,
+        dataType: "json"
+    }).responseJSON;
+
+    gpii.tests.flowManager.privacy.privacyToSettingsTransformRules = $.ajax({
+        type: "GET",
+        url: "../../../../../testData/ontologies/privacy-flat.json",
+        async: false,
+        dataType: "json"
+    }).responseJSON;
+
+    gpii.tests.flowManager.privacy.settingsToPrivacyTransformRules = fluid.model.transform.invertConfiguration(gpii.tests.flowManager.privacy.privacyToSettingsTransformRules);
+
+    fluid.defaults("gpii.tests.flowManager.privacy.privacyFilter", {
+        gradeNames: ["gpii.flowManager.privacy.privacyFilter", "autoInit"],
+        members: {
+            settingsToPrivacyTransformRules: gpii.tests.flowManager.privacy.settingsToPrivacyTransformRules,
+            privacyToSettingsTransformRules: gpii.tests.flowManager.privacy.privacyToSettingsTransformRules
+        }
+    });
+
+    gpii.tests.flowManager.privacy.testdata.tests = [
+        {
+            name: "No privacy settings",
+            privacySettings: { },
+            expected: {
+                "inferredConfiguration": {
+                    "gpii-default": {
+                        "applications": { }
+                    },
+                    "turn-down-light": {
+                        "applications": { },
+                        "conditions": [
+                            {
+                                "type": "http://registry.gpii.net/conditions/inRange",
+                                "min": 400,
+                                "inputPath": "http://registry\\.gpii\\.net/common/environment/visual\\.luminance"
```

Comment:
All instances of this term in trunk should have been changed to "illuminance" rather than "visual\\.luminance" - I think there are a couple of solutions in testData which are still lagging - we should fix these test fixtures so they aren't a source of the old term